app/models/edition.rb

```diff
@@ -30,6 +30,9 @@ class Edition < ActiveRecord::Base
   has_many :classification_featurings, inverse_of: :edition
   has_many :links_reports, as: :link_reportable
 
+  has_many :dependencies, class_name: 'EditionDependency', foreign_key: :edition_id, dependent: :destroy
```

Comment:
David, we want to reserve `has_many :edition_dependencies` to be used `as: :dependable` when we implement admin links, as explained with an example here: https://github.com/alphagov/whitehall/commit/5c04f8a2c3d025271f47841bad1b81cc4ddc24f3#commitcomment-10045409